### PR TITLE
Add other money

### PR DESCRIPTION
**Pull request by VincentDEJ**
Hi, i needed a lot of different currencies ! So i made it and share it with you :)
Is it ok ?

### CODE REVIEW

File: src/Z38/SwissPayment/Money/BHD.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    {
+        return 'BHD';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    final protected function getDecimals()
+    {
+        return 2;
```
Comment (z38) [inline src/Z38/SwissPayment/Money/BHD.php]: AFAIK, the Bahraini Dinar has 3 decimal places.

File: src/Z38/SwissPayment/Money/KWD.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    {
+        return 'KWD';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    final protected function getDecimals()
+    {
+        return 2;
```
Comment (z38) [inline src/Z38/SwissPayment/Money/KWD.php]: The Kuwaiti Dinar has 3 decimal places as well.

**Review by z38 — CHANGES_REQUESTED** [review]
Thank you for your contribution! Just two small remarks, after fixing those we're good to go.

**Comment by VincentDEJ** [comment]
it's updated ! ;)

**Review by z38 — APPROVED** [review]

**Comment by z38** [comment]
Thank you!

**Pull request merged**